README.md

```diff
@@ -0,0 +1,49 @@
+OSIC Reference implementation
+=============================
+
+# What is it?
+
+Reference implementation is opinionated way to deploy OpenStack. One of big
```

Comment:
Are we forcing our opinions upon deployers or making suggestions as to how we think they should build a cloud?